RecoLocalMuon/RPCRecHit/src/RPCRecHitStandardAlgo.cc

```diff
@@ -24,18 +25,32 @@ bool RPCRecHitStandardAlgo::compute(const RPCRoll& roll,
   const float fstrip = (roll.centreOfStrip(cluster.firstStrip())).x();
   const float lstrip = (roll.centreOfStrip(cluster.lastStrip())).x();
   const float centreOfCluster = (fstrip + lstrip)/2;
+  const double y = cluster.hasY() ? cluster.y() : 0;
+  Point = LocalPoint(centreOfCluster, y, 0);
```

Comment:
I don't follow.
Why is "y!=0" case important for errors and not important for position.
IMO, in fitting it's  more important to get the residual correct than the uncertainty